crates/bevy_render/src/color/mod.rs

```diff
@@ -436,13 +436,238 @@ impl Color {
         self
     }
 
-    /// Returns this color with blue set to a new value in sRGB colorspace.
+    /// Returns this color with blue set to a new value in sRGB colorspace, returned value is of variant [`Color::Rgba`].
     #[must_use]
     pub fn with_b(mut self, b: f32) -> Self {
         self.set_b(b);
         self
     }
 
+    /// Get the red in rgb linear colorspace.
+    pub fn rgb_linear_r(&self) -> f32 {
+        match self.as_rgba_linear() {
+            Color::RgbaLinear { red, .. } => red,
+            _ => unreachable!(),
+        }
+    }
+
+    /// Get the green in rgb linear colorspace.
+    pub fn rgb_linear_g(&self) -> f32 {
+        match self.as_rgba_linear() {
+            Color::RgbaLinear { green, .. } => green,
+            _ => unreachable!(),
+        }
+    }
+
+    /// Get the blue in rgb linear colorspace.
+    pub fn rgb_linear_b(&self) -> f32 {
+        match self.as_rgba_linear() {
+            Color::RgbaLinear { blue, .. } => blue,
+            _ => unreachable!(),
+        }
+    }
+
+    /// Set the red in rgb linear colorspace, `self` is modified to variant [`Color::RgbaLi and returnednear`].
```

Comment:
```suggestion
    /// Set the red in rgb linear colorspace, `self` is modified to variant [`Color::RgbaLinear`] and returned.
```
There are a few others with the same issue too.

Reply:
🤦‍♂️must have messed it up when doing find and replace